sources/packages/web/src/components/students/ApplicationDetails.vue

```diff
@@ -49,7 +49,10 @@ export default {
     const initialData = ref({} as StudentApplicationDetails);
     const dataForTracking = ref({} as StudentApplicationDetailsForTracking);
     const router = useRouter();
-    const customEventCallback = async (form: any, event: FormIOCustomEvent) => {
+    const customEventCallback = async (
+      _form: any,
```

Comment:
Why is the key starting with _? (_form). Is that a form.io expectation.?

Reply:
@dheepak-aot  It is a way to declare functions with multiple parameters and you do not really need all of them.

https://lookout.dev/rules/ignore-fist-parameter-of-function-if-unused-with-a-simple-underscore

Reply:
Got it. Thanks @henryoforeh-aot.